server/athenian/api/openapi/openapi.yaml

```
@@ -1031,6 +1059,7 @@ components:
       - date
       - values
     FilterItemsRequest:
+      description: PR filters for /filter/pull_requests.
```

`FilterItemsRequest` is being used to filter repos and contributors
```suggestion
      description: PR filters for /filter/contributors and /filter/repositories.
```